BioCro/DARPA/biomass_opti/scripts/optimize_biomass_coef.Rmd

```diff
@@ -0,0 +1,254 @@
+---
+title: "Optimizing Setaria Biomass Coefficients from BioCro v0.95"
+author: "Kristina Riemer, Jessica Guo"
+output: github_document
+urlcolor: blue
+editor_options: 
+  chunk_output_type: console
+---
+
+### Walkthrough for optimizing using `DEoptim`
+
+Optimizing biomass coefficients so that biomass estimates from BioCro as closely as possible match the measured biomass values. Parameters of interest are phenoParms, with thermal times and rhizome coefficients being fixed and leaf, stem, root, and grain coefficients being optimized across. This returns values of biomass for each stage and plant part, but the value being minimized is the difference between measured biomass and biomass estimated by BioCro. It seems like we can optimized across all stages at once within our objective function. 
```

Comment:
```suggestion
Optimizing biomass coefficients so that biomass estimates from BioCro as closely as possible match the measured biomass values. Parameters of interest are phenoParms, with thermal times and rhizome coefficients being fixed and leaf, stem, root, and grain coefficients being optimized across. This returns values of biomass for each stage and plant part, but the value being minimized is the difference between measured biomass and biomass estimated by BioCro. It seems like we can optimize across all stages at once within our objective function. 
```